lib/src/generated/cosmos/auth/v1beta1/genesis.pb.dart

```diff
@@ -33,31 +51,40 @@ class GenesisState extends $pb.GeneratedMessage {
     }
     return _result;
   }
-  factory GenesisState.fromBuffer($core.List<$core.int> i, [$pb.ExtensionRegistry r = $pb.ExtensionRegistry.EMPTY]) => create()..mergeFromBuffer(i, r);
-  factory GenesisState.fromJson($core.String i, [$pb.ExtensionRegistry r = $pb.ExtensionRegistry.EMPTY]) => create()..mergeFromJson(i, r);
-  @$core.Deprecated(
-  'Using this can add significant overhead to your binary. '
-  'Use [GeneratedMessageGenericExtensions.deepCopy] instead. '
-  'Will be removed in next major version')
+  factory GenesisState.fromBuffer($core.List<$core.int> i,
+          [$pb.ExtensionRegistry r = $pb.ExtensionRegistry.EMPTY]) =>
+      create()..mergeFromBuffer(i, r);
```

Comment:
same here, better to use more clear variables than "i", "r"

Reply:
IMO "i" for index is totally standard, and not a problem.

Reply:
Same auto generated code.